godot-google-play-billing/src/main/java/org/godotengine/godot/plugin/googleplaybilling/GodotGooglePlayBilling.java

```diff
@@ -68,7 +68,7 @@ public GodotGooglePlayBilling(Godot godot) {
 		super(godot);
 
 		billingClient = BillingClient
-								.newBuilder(getActivity())
+								.newBuilder(godot.requireContext())
```

Comment:
@m4gr3d thanks for the review! I don't get your point though 😅. How is the type of `godot` related to `requireContext`? I only need a Context here, not an Activity. Am I missing something?

Reply:
@timoschwarzer [`requireContext`](https://developer.android.com/reference/androidx/fragment/app/Fragment#requireContext()) (and `requireActivity` as well) are methods from the `Fragment` type and are only available because for `3.2.4`, the [`Godot` class was updated to extend from `Fragment` instead of `FragmentActivity`](https://github.com/godotengine/godot/pull/42186).
This change would actually cause the plugin to fail for users still on `3.2.3` as these methods would be missing.

`Activity` extends from `Context`, so using `getActivity()` and passing an `Activity` instance here is fine.

Reply:
Ohhhh I see. Thank you for the explanation. For some reason I thought `requireContext` etc. came from `Godot` :facepalm: . I'll change that.